vizro-ai/docs/pages/explanation/faq.md

```diff
@@ -22,23 +24,37 @@ We are working on supporting more models and more vendors. Stay tuned!
     Users should be cautious about sharing or inputting any personal or sensitive information.
 
     **Data is sent to model vendors if you connect to LLMs via their APIs.**
-    For example, if you specify model_name="gpt-3.5-turbo-0613", your data will be sent to OpenAI via their API.
+    For example, if you specify model="gpt-3.5-turbo-0613", your data will be sent to OpenAI via their API.
 
     Users are also recommended to review the third party API key section of the [disclaimer](../explanation/disclaimer.md) documentation.
 
 ## What parameters does Vizro-AI support?
 Currently, Vizro-AI supports the following parameters:
 
-- `temperature`: A parameter for tuning the randomness of the output. It is set to 0 by
-  default. We recommend setting it to 0 for Vizro-AI usage, as it's mostly
-  deterministic.
-- `model_name`: The name of the model to use. See above for [models currently supported by Vizro-AI](#which-llms-are-supported-by-vizro-ai) for the models supported.
+- `model`: The name of the model or the instantiated model to use. See above for [models currently supported by Vizro-AI](#which-llms-are-supported-by-vizro-ai) for the models supported.
```

Comment:
I have found another instance of `model_name` in one other guide - could you please double check that we have updated the entire documentation?